app/views/layouts/_boards_menu_section.html.erb

```diff
@@ -1,5 +1,20 @@
-<% boards.each do |board| %>
-  <li class="nav-item<%= board.id == @board.id ? ' active' : '' unless @board.nil? %>">
-    <%= link_to board.name, board_path(board), class: 'nav-link' %>
+<% if @boards.length > 5 %>
```

Comment:
Out of curiosity, how did you decide about 5 ? I mean on a small screen / large screen this limit could be different🙏

Reply:
Yeah it'd be preferable if this were dynamic/responsive. I'm not sure if that's easy with bootstrap. I selfishly used my screen (2880x1800 macbook) as a ruler :P.

Do you know if bootstrap can automate this for us? Otherwise the more manual (but still somewhat better way) may be to use something like:
```
em in px * # of characters in all board names combined + padding width
```
Which probably isn't the cleanest thing ever, but might work?

Reply:
Responsive would be great indeed. I was not asking for a custom solution or a solution at all and wanted to know how you decided about `5` 

Did you try to preview it with a different screen ? Might end up being better with 4. Anyway there will still be problems depending on number of chars in each piece (as you mentionned) but we can go gradually I guess 🤷‍♂️

Reply:
Yeah maybe it's better to switch it to 4 and open another issue to investigate the responsive stuff? Also depends on the board names, which is unfortunate. I could loop through and count characters maybe... which might be better (esp if we consider viewport width, even if it's not 100% calculated/the above formula)

Reply:
👌 Sounds good to me